cluster/manifests/skipper/crd.yaml

```diff
@@ -1,158 +1,209 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+
+---
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.0
+  creationTimestamp: null
   name: routegroups.zalando.org
 spec:
   group: zalando.org
-  version: v1
-  scope: Namespaced
   names:
-    kind: RouteGroup
-    singular: routegroup
-    plural: routegroups
     categories:
     - all
-    shortNames:
```

Comment:
@szuecs this unfortunately dropped the `shortNames`. I have made a fix here: https://github.com/szuecs/routegroup-client/pull/8

Reply:
Thanks, merge md it